content/de/about.md

```diff
@@ -14,6 +14,8 @@ Let's Encrypt ist eine freie, automatisierte und offene Zertifizierungsstelle (C
 
 Wir geben Menschen die digitalen Zertifikate, die sie zur Aktivierung von HTTPS (SSL/TLS) auf ihrer Webseite benötigen, kostenlos und in der benutzerfreundlichsten Variante, die uns möglich ist. Wir machen das, um ein noch sicheres Web zu erstellen, welches die Privatsphäre aktzeptiert.
 
+Lesen Sie unseren Jahresbericht des vergangenen Jahres durch Herunterladen der ([Desktop](https://abetterinternet.org/documents/2019-ISRG-Annual-Report-Desktop.pdf) oder die [Mobile](https://abetterinternet.org/documents/2019-ISRG-Annual-Report-Mobile.pdf)) Version.
```

Comment:
Can we improve this line even more?

```suggestion
Lesen Sie unseren Jahresbericht des vergangenen Jahres ([Desktop](https://abetterinternet.org/documents/2019-ISRG-Annual-Report-Desktop.pdf), [Mobil](https://abetterinternet.org/documents/2019-ISRG-Annual-Report-Mobile.pdf)).
```

Reply:
@max-re I've merged the PR as-is. If you think some things could be improved, do not hesitate to open a PR 🙂 Thanks again for you help!

Reply:
In fact you must download the report, there is no online reader. So the German translation is even better then the origin ;)